### PR TITLE
Fixes #22104 Fixing Findbug warnings 

### LINKED ISSUE #22104: Fix Findbugs warnings in GlassFish

### PR DESCRIPTION
**Pull request by mukumawa**
Fixes #22104

### CODE REVIEW

**Comment by mukumawa** [comment]
@glassfishrobot run ci tests please

**Comment by mukumawa** [comment]
@glassfishrobot run ci tests please

**Comment by mukumawa** [comment]
@glassfishrobot run ci tests please

**Comment by mukumawa** [comment]
@glassfishrobot run ci tests please

**Comment by yaminikb** [comment]
@glassfishrobot run ci tests pease

**Review by yaminikb — APPROVED** [review]
Looks fine

File: appserver/transaction/jts/src/main/java/com/sun/jts/CosTransactions/CoordinatorLog.java [hunk 1]
```diff
@@ -363,17 +363,9 @@ synchronized private void reUse(String logPath) {
...
             result = SectionPool.getCoordinatorLogSection(sectionName);
-            if( result == null ) {
-            }
-
-            // Copy in the name and set initial values of the other variables.
-
-            else {
-
                 // Add the new section information to the map.
+            sectionMapping.put(sectionName,result);
```
Comment (jayasheelankumar) [inline appserver/transaction/jts/src/main/java/com/sun/jts/CosTransactions/CoordinatorLog.java]: We can include this statement inside result != null condition.

File: appserver/transaction/jts/src/main/java/com/sun/jts/CosTransactions/CoordinatorLog.java [hunk 2]
```diff
@@ -1161,13 +1153,7 @@ private static boolean openLog() {
...
 			       CoordinatorLogPool.getCoordinatorLog());
 
-            if( logStateHolder.logFile == null ) {
-				_logger.log(Level.SEVERE,"jts.cannot_open_log_file",serverName);
-				 String msg = LogFormatter.getLocalizedMessage(_logger,
-				 						"jts.cannot_open_log_file");
-				 throw  new org.omg.CORBA.INTERNAL(msg);
-            } else 
-                Configuration.setLogFile(logStateHolder.logFile);
+            Configuration.setLogFile(logStateHolder.logFile);
```
Comment (jayasheelankumar) [inline appserver/transaction/jts/src/main/java/com/sun/jts/CosTransactions/CoordinatorLog.java]: Why is the complete if condition removed?

File: appserver/transaction/jts/src/main/java/com/sun/jts/CosTransactions/CoordinatorLog.java [hunk 3]
```diff
@@ -1222,13 +1208,7 @@ private static boolean openLog(String logPath) {
...
 			       CoordinatorLogPool.getCoordinatorLog(logPath));
 
-            if( logStateHolder.logFile == null ) {
-				_logger.log(Level.SEVERE,"jts.cannot_open_log_file",serverName);
-				 String msg = LogFormatter.getLocalizedMessage(_logger,
-				 						"jts.cannot_open_log_file");
-				 throw  new org.omg.CORBA.INTERNAL(msg);
-            } else 
-                Configuration.setLogFile(logPath,logStateHolder.logFile);
+            Configuration.setLogFile(logPath,logStateHolder.logFile);
```
Comment (jayasheelankumar) [inline appserver/transaction/jts/src/main/java/com/sun/jts/CosTransactions/CoordinatorLog.java]: Why is the complete if condition removed?

File: nucleus/admin/config-api/src/main/java/com/sun/enterprise/config/modularity/ConfigModularityUtils.java [hunk 4]
```diff
@@ -935,9 +934,6 @@ public String getRuntimeTypePrefix(StartupContext startupContext) {
             ConfigInjector injector = serviceLocator.getService(ConfigInjector.class, desc.getName());
             if (injector != null) {
                 String clzName = injector.getClass().getName().substring(0, injector.getClass().getName().length() - 8);
-                if (clzName == null) {
```
Comment (jayasheelankumar) [inline nucleus/admin/config-api/src/main/java/com/sun/enterprise/config/modularity/ConfigModularityUtils.java]: This condition is needed. If we remove this, the below statement will throw exception

File: nucleus/admin/config-api/src/main/java/com/sun/enterprise/config/modularity/GetSetModularityHelper.java [hunk 5]
```diff
@@ -66,47 +66,6 @@
     private Domain domain;
 
     /**
-     * checks and see if a class has an attribute with he specified name or not.
```
Comment (jayasheelankumar) [inline nucleus/admin/config-api/src/main/java/com/sun/enterprise/config/modularity/GetSetModularityHelper.java]: Are these methods unused?

File: nucleus/admin/config-api/src/main/java/com/sun/enterprise/config/serverbeans/Server.java [hunk 6]
```diff
@@ -598,10 +598,9 @@ public void decorate(AdminCommandContext context, final Server instance) throws
                     List<ServerRef> serverRefs = c.getServerRef();
                     Iterator<ServerRef> serverRefIter = serverRefs.iterator();
                     boolean allLBEnabled = false;
-                    while (!allLBEnabled && serverRefIter.hasNext()) {
```
Comment (jayasheelankumar) [inline nucleus/admin/config-api/src/main/java/com/sun/enterprise/config/serverbeans/Server.java]: Are we sure if this condition is not required? In the while loop the value of allLBEnabled is being modified.

File: nucleus/admin/util/src/main/java/com/sun/enterprise/admin/remote/sse/GfSseEventReceiver.java [hunk 7]
```diff
@@ -89,7 +89,7 @@ public GfSseInboundEvent readEvent() throws IOException {
                         } else if(data != '\n') {
                             baos.write(data);
                             currentState = State.FIELD_NAME;
-                        } else if(data == '\n') {
```
Comment (jayasheelankumar) [inline nucleus/admin/util/src/main/java/com/sun/enterprise/admin/remote/sse/GfSseEventReceiver.java]: Are we sure that removing of the If condition here is not causing any issue? I think the logic changes when we remove the condition

**Pull request merged**